src/components/message/message-card/MessageBodyCard.tsx

```diff
@@ -99,6 +128,15 @@ function MessageBodyCardField(props: FieldProps) {
 		);
 	}
 
+	const subFields =
+		!isSimpleValue(field) && !isListValue(field)
```

Comment:
lets create helper function isMessageValueField

we can also memoize `sortedSubFields` since we memoized it in MessageBodyCard